ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -117,31 +117,58 @@ class LocalAssemblerData
         }
     }
 
-    Eigen::Map<const Eigen::RowVectorXd>
-    getShapeMatrix(const unsigned integration_point) const override
+    /// Computes the flux in the point \c p_local_coords that is given in local
+    /// coordinates using the values from \c local_x.
+    std::vector<double> getFlux(
+        MathLib::Point3d const& p_local_coords,
+        std::vector<double> const& local_x) const override
+    {
+        // eval dNdx and invJ at p
+        using FemType =
+            NumLib::TemplateIsoparametric<ShapeFunction, ShapeMatricesType>;
+
+        FemType fe(*static_cast<const typename ShapeFunction::MeshElement*>(
+            &_element));
+
+        typename ShapeMatricesType::ShapeMatrices shape_matrices(
+            ShapeFunction::DIM, GlobalDim, ShapeFunction::NPOINTS);
+
+        fe.computeShapeFunctions(p_local_coords.getCoords(), shape_matrices,
+                                 GlobalDim);
+        std::vector<double> gradient;
+        gradient.resize(3);
+        Eigen::Map<Eigen::RowVectorXd>(gradient.data(), gradient.size()) =
```

Comment:
Until know for the simple tests conductivity was always 1. I'll add conductivity to the formula.